MetaMask/metamask-extension
# Remove textarea caret dep

## Explanation

At the screen where the user inserts their password, every keystroke that occurs triggers textarea-caret dependency
If textarea-caret is breached it can be bad, a third party package should not be triggered in such a sensitive phase.
This fix replaces this dependency with our own less generic implementation
Fixes Logo head following password as it is being typed
Fixes  [cirt:#114](https://github.com/MetaMask/pm-security/issues/114)

## Manual Testing Steps

lock metamask, start typing in password, verify that the logo head is following latest character correctly.
verify different browsers and resolutions


## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
